packages/gamut-system/src/core/system/index.ts

```diff
@@ -56,7 +57,8 @@ const create = <
 
     // Return the variant function
     return (props: any) => {
-      const variantProps = variants[props[propKey]] || {};
+      const variantProps =
+        variants[props[propKey]] || variants[defaultKey] || {};
```

Comment:
⛳  since we can spread undefined things safely

```suggestion
const variantProps = variants[props[propKey] || defaultKey];
```

Reply:
I had no idea we could spread undefined, nice